instrumentation/elasticsearch/elasticsearch-transport-5.0/javaagent/elasticsearch-transport-5.0-javaagent.gradle

```diff
@@ -5,12 +5,20 @@ muzzle {
     group = "org.elasticsearch.client"
     module = "transport"
     versions = "[5.0.0,5.3.0)"
+    // TODO (trask) try to remove the skipVersions in the future
+    //  not sure what's wrong with this recent version, hopefully just publishing blip that
+    //  will work itself out
+    skipVersions += ['7.11.0']
```

Comment:
But we already limit muzzle with `5.3.0)`. Why this is needed?

Reply:
It's needed because of `assertInverse = true` - this creates a `fail` task for all other versions, 7.11.0 included.